### PR TITLE
Move command constructor args to `handle()`

### PR DESCRIPTION
**Pull request by stevebauman**
Closes https://github.com/babenkoivan/elastic-migrations/issues/30

I tried to match your code style as best I could when updating the tests, but they may need some adjustment to suit your preferences.

Let me know if you have any comments/questions/concerns.

No hard feelings on closure ❤️

### CODE REVIEW

**Comment by stevebauman** [comment]
The updated tests fail static analysis due to PHPUnit's mock objects. Should we set the analyzer to ignore the `tests/*` folder?

File: tests/Integration/Console/StatusCommandTest.php [hunk 1]
```diff
@@ -29,7 +30,7 @@ protected function setUp(): void
 
         $this->migrator = $this->createMock(Migrator::class);
```
Comment (babenkoivan) [inline tests/Integration/Console/StatusCommandTest.php:31]: Hey, can we revert the changes in the tests and just help Laravel to resolve the dependencies as follows:

```php
$this->migrator = $this->createMock(Migrator::class);
$this->app->instance(Migrator::class, $this->migrator);
```

i.e. all we need to do is to register mocked objects in the service container, this will fix the phpstan issue.

**Comment by babenkoivan** [comment]
> The updated tests fail static analysis due to PHPUnit's mock objects. Should we set the analyzer to ignore the `tests/*` folder?

I think we just need to write the tests a bit differently, I left a comment with an example.

**Comment by stevebauman** [comment]
Great idea @babenkoivan 👍 , I've reset the tests and have done what you've requested. We're now all green.

**Review by babenkoivan — APPROVED** [review]

**Comment by babenkoivan** [comment]
Thank you @stevebauman! I've just released [v1.6.2](https://github.com/babenkoivan/elastic-migrations/releases/tag/v1.6.2), which includes your changes 🎉

**Comment by stevebauman** [comment]
Awesome, happy to help @babenkoivan! Appreciate it! 👍  Thanks for pushing it to a new release 🎉
